services/bots/src/github-webhook/handlers/platinum_review.ts

```diff
@@ -76,7 +76,7 @@ export class PlatinumReview extends BaseWebhookHandler {
           ? `Code owner approval required before merging`
           : `Code owner approval ok.`,
         target_url:
-          'https://developers.home-assistant.io/docs/integration_quality_scale_index/#platinum-',
+          'https://developers.home-assistant.io/docs/integration_quality_scale_index/#-platinum',
```

Comment:
Is this link not outdated completely?

Reply:
It redirects to https://developers.home-assistant.io/docs/core/integration-quality-scale/#-platinum 
Should I replace it with the final one?

Reply:
I would pick the final one yes